packages/apollo-language-server/src/config/config.ts

```diff
@@ -162,12 +162,12 @@ export class ApolloConfig {
     return configs;
   }
 
-  set tag(tag: string) {
-    this._tag = tag;
+  set variant(tag: string) {
```

Comment:
My only concern is if this breaks anything in vs code. But I don’t think it should

Reply:
I admit I didn't test with VS code, but mainly because I don't know how to test a local build! Is there a QA practice we go through before release?

Reply:
Not in a consistent way! I’ll run through all this before releasing, but I don’t expect any issue here since it only interfaces with the language server anyway :)